omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -360,16 +360,6 @@
             Source="$(var.EtcFiles)\datadog.yaml.example"/>
       </Component>
 
-      <Component Id="install_info"
-                 Guid="260CEE46-E9E9-4EEF-A065-969524970B68"
-                 Permanent="no">
-        <Condition>
-          <![CDATA[UILevel > 2]]>
-        </Condition>
-        <File Id="install_info"
-            Name="install_info"
-            Source="Resources\assets\install_info"/>
-      </Component>
```

Comment:
We want to continue doing this old logic if OVERRIDE_INSTALLATION_METHOD isn't set.